node/node.go

```diff
@@ -130,6 +133,82 @@ func StateProvider(stateProvider statesync.StateProvider) Option {
 	}
 }
 
+// This function is expected to synchronize CometBFTs
+// stores with the application after statesync has been performed offline
+// It is expected that the blockstore and statestore are empty at
+// the time the function is called
+func BootstrapState(height uint64, appHash []byte, config *cfg.Config) error {
+	ctx := context.Background()
+	logger := log.NewTMLogger(log.NewSyncWriter(os.Stdout))
+	if config == nil {
+		config = cfg.DefaultConfig()
+	}
+	blockStore, stateDB, err := initDBs(config, cfg.DefaultDBProvider)
```

Comment:
Are the SDK folks OK with `cfg.DefaultDBProvider` hardcoded here, or do they prefer to pass it as a parameter?

Reply:
Added it as a parameter. Although currently both in our code and in the SDK code it is the DefaultDBProvider.